src/components/contact-item/contact-item.ts

```diff
@@ -1,15 +1,35 @@
 import Block from '../../core/Block';
+import imagePlaceholder from '../../img/contact/contact-1.png';
+
+interface ContactItemProps {
+  id: string;
+  image: string;
+  name: string;
+  text: string;
+  date: string;
+  unread: string;
+  onClick: () => void;
+}
+
 export class ContactItem extends Block {
 
+  constructor({id, image, name, text, date, unread, onClick}: ContactItemProps) {
+    super({id, image, name, text, date, unread, events: {click: onClick}});
```

Comment:
Можно немного сократить
```suggestion
  constructor({...props, onClick}: ContactItemProps) {
    super({...props, events: {click: onClick}});
```